src/main/java/soonkeatneo/duke/TaskList.java

```diff
@@ -0,0 +1,194 @@
+package soonkeatneo.duke;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+import soonkeatneo.duke.task.Deadline;
+import soonkeatneo.duke.task.Event;
+import soonkeatneo.duke.task.Task;
+import soonkeatneo.duke.task.Todo;
+
+/**
+ * Stores and manages the list of {@Task} created.
+ * @author Soon Keat Neo
+ * @version CS2103T AY20/21 Sem 2 iP v0.1
+ */
+public class TaskList {
+    private static List<Task> taskList;
+
+    /**
+     * Creates a new task list without reading from existing storage.
+     */
+    public TaskList() {
+        this.taskList = new ArrayList<>();
+    }
+
+    /**
+     * Overloaded constructor creating a new task list reading from existing {@Storage}.
+     * @param storage Storage object to be read from
```

Comment:
I believe a line should be spaced between the description and the @param for all JavaDoc commentary (even though it is not explicitly stated in this mod's check requirements). The JavaFX files practice this too.